velox/docs/develop/types.rst

```diff
@@ -73,6 +73,31 @@ Examples:
 * Timestamp(-5000*24*60*60 - 1000, 123456) represents 1956-04-24 07:43:20.000123456
   (5000 days 1000 seconds before epoch plus 123456 nanoseconds).
 
+Floating point types (REAL, DOUBLE) have special values negative infinity, positive infinity, and
+not-a-number (NaN).
+
+For NaN the semantics are different than the C++ standard floating point semantics:
```

Comment:
Maybe mention that we treat all different NaNs (+/-, quiet/signaling) as a canonical NaN (+, quiet)